templates/projects/classic/README.md

```diff
@@ -0,0 +1,139 @@
+
+
+## Develop your project
+
+This section provides commands that you will frequently use during development.
+
+
+### Initial build
+
+```shell
+make install
+```
+
+
+### Start servers
+
+Start the backend server with the following command.
+
+```shell
+make backend-start
+```
+
+
+### Rebuild after changes
+
+After you make changes to your code, you will need to install the changes and restart the server.
+
+```shell
+make install
+make backend-start
+```
+
+
+## Project generation options 🛠️
+
+The table below describes the options you can customize using the [Cookiecutter CLI](https://github.com/cookiecutter/cookiecutter) during the generation process.
+
+| Option                | Description                                                                                                                                          | Example                       |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| `title`       | Your project's human-readable name, capitals and spaces allowed.                                                                                     | `Plone Site`                |
+| `description`         | Describes your project and gets used in places like ``README.md`` and such.                                                                          | `New site for our company.` |
+| `project_slug`        | Your project's slug without spaces. Used to name your repository and Docker images.                                                                  | `plone-site`                |
+| `hostname`            | Hostname where the project will be deployed.                                                                                                         | `site.plone.org`            |
+| `author`              | This is you! The value goes into places like ``LICENSE``, ``setup.py`` and such.                                                                     | `Our Company`               |
+| `email`               | The email address you want to identify yourself in the project.                                                                                      | `email@example.com`         |
+| `use_prerelease_versions`  | Use non-stable versions of Plone, (The default value could also be set via `USE_PRERELEASE` environment variable.                                   | `Yes`                       |
+| `plone_version`       | Plone version to be used. This queries for the latest available Plone 6 version and presents it to you as the default value.                         | `6.0.0`                     |
+| `python_package_name` | Name of the Python package used to configure your project. It needs to be Python-importable, so no dashes, spaces or special characters are allowed. | `plone_site`                |
```

Comment:
Please check this statement for accuracy. It's more explicit to say what is valid, instead of what is not valid.

```suggestion
| `python_package_name` | Name of the importable Python package used to configure your project. A valid name consists only of ASCII letters and numbers, period, and underscore. It must start with a letter, and end with a letter or number. | `plone_site`                |
```